packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -93,49 +93,60 @@ const generateAltButton = (color: string) => {
 
 const $spinnerSize = cssVar('spinner-size');
 
+const iconSizes = {
+  xs: '0.75rem',
+  s: '1rem',
+  m: '1.25rem',
+  l: '1.25rem',
+};
```

Comment:
this has to use the Icon's size, not duplicate the values

Reply:
fixed